### PR TITLE
Fixes #2578 -NTR- fibroblastic reticular cell

### LINKED ISSUE #2578: [NTR] intestinal fibroblastic reticular cell
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

**Preferred term label**
intestinal fibroblastic reticular cell

**Synonyms** (add reference(s), please)
FRCs (related synonym) [doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)
solitary intestinal lymphoid tissue (SILT) FRCs (narrow synonym) [doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)
gut-associated secondary lymphoid organs (SLOs)  FRCs (related synonym) [doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)
Ccl19-expressing FRCs (narrow synonym) [doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)
Ccl19+ fibroblastic reticular cells PMID: [36405734](https://pubmed.ncbi.nlm.nih.gov/36405734)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A specialized fibroblastic reticular cel that is part of gut-associated lymphoid tissues(GALT), responsible for forming a structural network that facilitates immune cell interactions. These cells are found in Peyer's patches, cryptopatches, and isolated lymphoid follicles. They play crucial roles in maintaining intestinal immunity by controlling innate lymphoid cell homeostasis and function, organizing lymphoid structures, and contributing to intestinal microbiome balance.

[doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)
PMID: [33707780](https://pubmed.ncbi.nlm.nih.gov/33707780)
[doi:/10.1038/s41467-022-29734-2](https://www.nature.com/articles/s41467-022-29734-2)

**Extended**

Intestinal fibroblastic reticular cells (FRCs) develop from both perivascular and subepithelial progenitors during Peyer's patch formation. Intestinal fibroblastic reticular cells express key markers such as CCL19, CCL21, CXCL13, TNFSF13B (also known as BAFF), and produce IL-7, which are important for their immune regulatory functions and the organization of lymphoid structures.
PMID: [33707780](https://pubmed.ncbi.nlm.nih.gov/33707780)

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[fibroblastic reticular cell](http://purl.obolibrary.org/obo/CL_0009101)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
part of [gut-associated lymphoid tissue](http://purl.obolibrary.org/obo/UBERON_0001962)?

**Your ORCID**

**Additional notes or concerns**

### PR DESCRIPTION
**Pull request by Caroline-99**
- Added a new term
- Added textual and logical definition with extended description
- Added synonyms
- Added references

### CODE REVIEW

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32470,6 +32471,20 @@ AnnotationAssertion(rdfs:label obo:CL_4052003 "intestinal villus capillary endot
...
+AnnotationAssertion(terms:contributor obo:CL_4052004 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052004 "2024-09-19T11:22:18Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33707780") Annotation(oboInOwl:hasDbXref "doi:/10.1038/s41467-022-29734-2") terms:description obo:CL_4052004 "Intestinal fibroblastic reticular cells (FRCs) develop from both perivascular and subepithelial progenitors during Peyer's patch formation. Intestinal fibroblastic reticular cells express key markers such as CCL19, CCL21, CXCL13, TNFSF13B (also known as BAFF), and produce IL-7, which are important for their immune regulatory functions and the organization of lymphoid structures.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:/10.1038/s41467-022-29734-2") oboInOwl:hasNarrowSynonym obo:CL_4052004 "Ccl19-expressing FRCs")
+AnnotationAssertion(Annotation(rdfs:comment "doi:/10.1038/s41467-022-29734-2") oboInOwl:hasNarrowSynonym obo:CL_4052004 "solitary intestinal lymphoid tissue (SILT) FRCs")
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:32481]: What's wrong with abbvs in synonyms?  Didn't realise we had a policy against that.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32470,6 +32471,20 @@ AnnotationAssertion(rdfs:label obo:CL_4052003 "intestinal villus capillary endot
 EquivalentClasses(obo:CL_4052003 ObjectIntersectionOf(obo:CL_0002144 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001213)))
 SubClassOf(obo:CL_4052003 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0002064))
 
+# Class: obo:CL_4052004 (intestinal fibroblastic reticular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33707780") Annotation(oboInOwl:hasDbXref "doi:/10.1038/s41467-022-29734-2") obo:IAO_0000115 obo:CL_4052004 "A specialized fibroblastic reticular cell that is part of gut-associated lymphoid tissues(GALT), responsible for forming a structural network that facilitates immune cell interactions. This cell is found in Peyer's patches, cryptopatches, and isolated lymphoid follicles. It plays crucial roles in maintaining intestinal immunity by controlling innate lymphoid cell homeostasis and function, organizing lymphoid structures, and contributing to intestinal microbiome balance.")
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:32476]: I agree that abbreviations should not be used on there own, but I don't see a problem with including the abbreviation in brackets after the full terms.   The vast majority of the literature uses GALT, so I think important for users to see this.

File: src/ontology/cl-edit.owl (hunk 1 above)
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:32481]: Abbreviations are fine, however I was told that they should be a separate synonym with the AP has_synonym_type 'abbreviation.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**